types/archiver/index.d.ts

```diff
@@ -18,30 +18,33 @@
 import * as FS from 'fs';
 import * as STREAM from 'stream';
 
-
-declare function archiver(format: string, options?: archiver.Options): archiver.Archiver;
+declare function archiver(format: string, options?: {}): archiver.Archiver;
```

Comment:
Should the type of options be "CoreOptions | TransformOptions" (as indicated here https://archiverjs.com/docs/Archiver.html)